.env

```diff
@@ -1,13 +1,13 @@
 VERSION=0.32.11-alpha
 
 # Airbyte Internal Job Database, see https://docs.airbyte.io/operator-guides/configuring-airbyte-db
-DATABASE_USER=docker
-DATABASE_PASSWORD=docker
+JOBS_DATABASE_USER=docker
```

Comment:
should be `JOB_` everywhere for the database, not a mix of `JOB_` and `JOBS_`.

Reply:
yeap, I will come back to this. abandoned since I need to focus on getting the server HA out and most of the renaming has already been done.